src/options.js

```diff
@@ -3,27 +3,54 @@
 const CATEGORY_PHP = "PHP";
 
 module.exports = {
+  phpVersion: {
+    since: "0.13.0",
+    category: CATEGORY_PHP,
+    type: "choice",
+    default: "7.1",
+    description: "Minimum target PHP version.",
+    choices: [
+      { value: "5.0" },
+      { value: "5.1" },
+      { value: "5.2" },
+      { value: "5.3" },
+      { value: "5.4" },
+      { value: "5.5" },
+      { value: "5.6" },
+      { value: "7.0" },
+      { value: "7.1" },
+      { value: "7.2" },
+      { value: "7.3" },
+      { value: "7.4" }
+    ]
+  },
   trailingCommaPHP: {
     since: "0.0.0",
     category: CATEGORY_PHP,
     type: "choice",
-    default: "none",
+    default: "all",
```

Comment:
I didn't start with a Boolean because of the missing deprecation story - but if we don't expect a third value for this in the future then changing to a Boolean is probably the best way forward. I can't imagine a third option value for this, so I'm in :+1:

Reply:
Done. I'd say we wait with changing the README until we're ready to release 0.13.0.

Reply:
Alright, cool. Also agree with default to `true`.